plugins/sudo/sudo.plugin.sh

```diff
@@ -1,11 +1,21 @@
 #! bash oh-my-bash.module
 
 function _omb_plugin_sudo_add_sudo {
+  function __omb_plugin_sudo_is_inserted {
```

Comment:
Can you put the function outside? Also, can you rename it as `_omb_plugin_sudo_is_inserted`, or `_omb_plugin_sudo__is_inserted` if you would like to explicitly make it *private*. cf #280

Reply:
Ok, I'll rename, I think that if it keep there it can decrease the speed too

Reply:
Is there a reason to put the function inside the function?

Bash doesn't support local functions. If one defines the function inside the function, the function will be defined dynamically in the global space and remains after the outer function is returned. With the current implementation, the function `__omb_plugin_sudo_is_inserted ` is newly defined in the global space overwriting the previous one every time the function `_omb_plugin_sudo_add_sudo` is called.

Reply:
It was just to looks like a local function, just trash of my head